server/src/infra/repositories/media.repository.ts

```diff
@@ -27,10 +35,18 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
+    const orientation = options.orientation;
+    const rotation =
+      options.format === 'webp'
+        ? orientation
+          ? orientationExifToDegrees(orientation.toString() as Orientation)
+          : undefined
+        : undefined;
     await sharp(input, { failOn: 'none' })
       .pipelineColorspace(options.colorspace === Colorspace.SRGB ? 'srgb' : 'rgb16')
       .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-      .rotate()
+      .rotate(rotation)
```

Comment:
Is this change required because it needs to use the overridden value from the sidecar file now? (as opposed to the value in the original file itself?

Reply:
Yes, exactly